### ISSUE TITLE
ICY / Essent partnership will end

### ISSUE THREAD

**Issue by mettin** [kerkenit/nl.marcovantklooster.icyethermostaat#5]
https://www.essent.nl/content/particulier/energie-besparen/smart-home/slimme-thermostaat/e-thermostaat.html#
Will the thermostat still work after this deadline without the subscription?

**Comment by kerkenit** [comment]
No, it won't. It uses the same cloud. I've installed a new smart heater (Tado) because of this.
So I can't continue development on this app. So probably at Homey 2.0 final release it won't work at all. Even if you buy a subscription, because the app isn't compatible with Homey 2.0 an I can't develop anymore

**Comment by mettin** [comment]
Okay thanks. Will upgrade thermostat than also. Thanks for the great app though!
